index.js

```diff
@@ -93,6 +93,7 @@ module.exports = function (babel) {
 
         let modules = state.opts.modules || {
           'htmlbars-inline-precompile': 'default',
+          'ember-cli-htmlbars': 'hbs',
```

Comment:
https://github.com/ember-cli/babel-plugin-htmlbars-inline-precompile/issues/292 claims that `ember-cli-htmlbars` currently is not supported but due to the `state.opts.modules ||` part of this statement I'm wondering if that's really the case. what's stopping you from passing in a `modules` option from the outside?

Reply:
I wanted to also throw this error and make sure tests passes with correct imports on both cases: https://github.com/ember-cli/babel-plugin-htmlbars-inline-precompile/pull/293#discussion_r520162431 . Also this results in better defaults, allows any developer to use it without passing that `opts`, which makes sense because that really should be the default behaviour practically speaking.

Reply:
> what's stopping you from passing in a modules option from the outside?

FWIW, I mentioned this somewhere else, but that is exactly what we are doing in ember-cli-htmlbars

https://github.com/ember-cli/ember-cli-htmlbars/blob/df8ed20cfa06634747bae3098dd292e8d03de49c/lib/utils.js#L11-L15

Either way, it seems totally fine to have this package default the configuration to match what we expect folks to actually use in real life.